libs/web/certificate-issuer/src/lib/certificate.service.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { environment } from '@caas/web/env';
```

Comment:
Same problem in 'artifact-runner', this creates a circular dependency we need to create a extra api module with all services and maybe an extra configuration module

Reply:
Will be changed in a future PR (see #15.)